.github/workflows/deploy-docs.yml

```
@@ -56,6 +66,27 @@ jobs:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    steps:
+      - name: Deploy
+        id: deployment
+        uses: actions/deploy-pages@v4
+
+
+
+
+
+ns/upload-pages-artifact@v3
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 86: could not find expected ':'

(syntax-check)